Makefile

```diff
@@ -192,6 +193,18 @@ push: push-datamgr push-plugin
 push-pp:
 	$(MAKE) push-plugin LOCALMODE=true VERSION=$(VERSION)-pp
 
+QUALIFIED_TAG ?=
+RELEASE_TAG ?= latest
+release:
```

Comment:
The default `REGISTRY` , dpcpinternal, is compiled into the binary of the original image via the go linker flag. Re-tagging may not be able to change the compiled go linker flag. 

To verify the theory above, please run the following steps to double check.
1. run `make release ...`
2. deploy the new image into your cluster and check if the image used data manager pod is also from vsphereveleroplugin.